solidity/contracts/system/TBTCSystem.sol

```diff
@@ -590,23 +594,58 @@ contract TBTCSystem is Ownable, ITBTCSystem, DepositLog {
     }
 
     /// @notice Request a new keep opening.
-    /// @param _m Minimum number of honest keep members required to sign.
-    /// @param _n Number of members in the keep.
+    /// @param _lotSizeSatoshis Lot size in satoshis.
     /// @param _maxSecuredLifetime Duration of stake lock in seconds.
     /// @return Address of a new keep.
     function requestNewKeep(
-        uint256 _m,
-        uint256 _n,
-        uint256 _bond,
+        uint64 _lotSizeSatoshis,
```

Comment:
Well now I feel weird about this taking the lot size 🤔 

I think it's a marked improvement though.

Reply:
Hmm, it feels quite natural to me but I don't have the full context. `Deposit` is initialized with the lot size and `TBTCSystem` does want to work with supported lot sizes. The moment we open a keep is where lot sizes ends and wei bonds come into play.

Reply:
It feels like by the time we reach the concept of a keep, we should be requesting the keep with a particular bond for the keep, rather than passing the lot size. But I like that the price feed and bond calculation is here now. So I'm still a little ways away from figuring out what's bothering me about it :) Regardless, I don't think it's worth doing anything more here for now.